sklearn/tests/metadata_routing_common.py

```diff
@@ -197,7 +201,9 @@ def __init__(self, registry=None, alpha=0.0):
         self.alpha = alpha
         self.registry = registry
 
-    def partial_fit(self, X, y, sample_weight="default", metadata="default"):
+    def partial_fit(
+        self, X, y, classes=None, sample_weight="default", metadata="default"
```

Comment:
if you're adding `classes` here, it should also be passed to `record_metadata_not_default` bellow, and the default value should be `"default"`, and `self.classes_` should be set to this value if this is not the default or `None`

Reply:
I don't think I'm wrong by stating that for classifier, we always have `classes` in `partial_fit` signature?

Reply:
so my comment is void, since we're not _routing_ classes here.

As for `classes`, in so many cases our estimators don't accept `classes`, do a `git grep "def partial_fit"`, it's kinda bad.

Reply:
Yes but the one not requested `classes` are transformers or regressors and it makes sense :).